.babelrc

```diff
@@ -0,0 +1,18 @@
+{
+    "presets": [
+        [
+            "@babel/preset-env",
+            {
+                "targets": {
+                    "node": "8"
```

Comment:
Do we still want 8? (I guess we're still 8 for dev environments?)

Reply:
Yeah as I noted in the PR description, we're still running node 8 for devs, so that pins us to 8 for now.